src/UhdmAst.cpp

```diff
@@ -860,12 +869,16 @@ namespace UhdmAst {
 
           // Get parameter assignments
           itr = vpi_iterate(vpiParamAssign, obj_h);
+          std::map<std::string, AstNode*> parameter_map;
           while (vpiHandle vpi_child_obj = vpi_scan(itr) ) {
             vpiHandle param_handle = vpi_handle(vpiLhs, vpi_child_obj);
             std::string param_name = vpi_get_str(vpiName, param_handle);
             sanitize_str(param_name);
             UINFO(3, "Got parameter (pin) " << param_name << std::endl);
             auto is_local = vpi_get(vpiLocalParam, param_handle);
+            std::string is_imported;
+            if (auto s = vpi_get_str(vpiImported, param_handle))
```

Comment:
This `if` repeats a few times in this PR. Consider making it a separate function. Also, `is_imported` sounds like a `bool`, I think it could be better named.

Reply:
Moved to a separate function.